Dockerfile

```diff
@@ -1,16 +1,24 @@
-FROM golang:1.9.6 AS build
+FROM golang:1.10-alpine AS build
 ADD . /src
 WORKDIR /src
-RUN go get -d -v -t
-RUN go test --cover ./... --run UnitTest
-RUN go build -v -o docker-flow-proxy
+RUN set -x \
```

Comment:
why set -x?

Reply:
Because the commands are chained together into one large command it is somtimes hard to figure which of those provoked an error occurred in the build process. The `set -x` turns the tracing functionality on so before execution of each individual command the command will be printed as text. This is also common practise in a lot of official Dockerfiles, for example: [MariaDB](https://github.com/docker-library/mariadb/blob/4891ee2e3bd2dc6b07db634a39433ad579764a4b/10.3/Dockerfile#L8) or [Nextcloud](https://github.com/nextcloud/docker/blob/1654a164dc1a7f6e01f418dcf8b5e2c48f08f708/13.0/fpm/Dockerfile#L5)